addons/web/static/src/js/core/widget.js

```diff
@@ -368,6 +368,29 @@ var Widget = core.Class.extend(mixins.PropertiesMixin, ServicesMixin, {
             this.$el.hasClass('o_hidden') ? this.do_show() : this.do_hide();
         }
     },
+    /**
+     * Enables the buttons of the widget
+     */
+    do_enable_buttons: function () {
```

Comment:
Not sure you want to add these methods in all widgets.
Why don't you just put them in the chatter widget?